app/controllers/albums_controller.rb

```diff
@@ -66,4 +68,8 @@ def set_album
     def album_params
       params.require(:album).permit(:post_id, images: [])
     end
+
+    def album_params_no_images
```

Comment:
I think this is not a meanigful abstraction in the domain.
We should try to achive it in some other way.